common/src/main/java/io/github/gaming32/bingo/data/goal/GoalManager.java

```diff
@@ -17,14 +18,15 @@
 import java.util.Map;
 import java.util.Set;
 
+// FIXME: Broken on Fabric until https://github.com/FabricMC/fabric/pull/4180 is merged
```

Comment:
Is this relating to goal loading entirely or just the reloading? If the former this is a pretty serious issue that we should probably find a workaround for

Reply:
This is related to goal loading entirely. I will workaround it if it isn't fixed in Fabric soon.